flood_adapt/object_model/scenario.py

```diff
@@ -73,6 +76,55 @@ def run(self):
                 f"Direct impacts for scenario '{self.attrs.name}' has already been run."
             )
 
+        # Create the infometrics and infographic files
+        self._create_infometrics_infographics()
+
+    def _create_infometrics_infographics(self):
```

Comment:
wouldn't it make more sense to have two seperate methods for the creation of the metrics table and the creatio of the infographic?